arduino/arduino.ino

```diff
@@ -0,0 +1,165 @@
+#include "DHT.h"
+#include <SoftwareSerial.h>
+
+#define DHTPIN 12
+#define DHTTYPE DHT11
+#define LEDgreen 10
+#define LEDyellow 9
+#define LEDred 8
+
+SoftwareSerial co2Serial(2, 3); // define MH-Z19 RX TX
+DHT dht11HNBK(DHTPIN, DHTTYPE);
+
+void setup() {
+  Serial.begin(9600);
+  pinMode(LEDgreen,OUTPUT);
+  pinMode(LEDyellow,OUTPUT);  
+  pinMode(LEDred,OUTPUT);
+    
+  co2Serial.begin(9600);
+  dht11HNBK.begin();
+
+}
+
+void loop() {
+  delay(5000);
+  
+  float hudty = dht11HNBK.readHumidity();
+  float temp = dht11HNBK.readTemperature();
+  
+  if(isnan(hudty) || isnan(temp))
+  {
+    Serial.println("Ein Fehler ist bei der Datenübertragung aufgetreten. \nDie Sensordaten konnten nicht ausgelesen werden. Bitte versuchen Sie es erneut!");
+    return;
+  }
+  
+  int ppm = 0, temperature = 0;
+  readSensor(&ppm, &temperature);
+  
+  if (ppm < 1000 && (40 > hudty && hudty < 60 ) && temp < 26) // gute Luft
```

Comment:
Da ist bei hudty noch ein Fehler in der Abfrage

Reply:
Danke dir für den Hinweis, ich gehe der Sache gerne nach :D